presto-geospatial/src/main/java/com/facebook/presto/plugin/geospatial/GeoFunctions.java

```diff
@@ -1681,4 +1785,41 @@ public OGCGeometry next()
             return geometriesDeque.pop();
         }
     }
+
+    private static class CartesianPoint
+    {
+        public double x;
+        public double y;
+        public double z;
+
+        public CartesianPoint(Point p) {
+            // Angle from North Pole down to Latitude, in Radians
+            double phi = toRadians(90 - p.getX());
```

Comment:
Good catch, I reviewed https://prestodb.io/docs/current/functions/geospatial.html and how Presto uses (long, lat) instead of (lat,long). will make the change